### ISSUE TITLE
No support for tracking results during training (tm.fit())

### ISSUE THREAD

**Issue by OleGunvaldsen** [cair/tmu#10]
There is no support for tracking loss/accuracy during training and testing of the tsetlin machine. The only data that is available is the end results. It would be helpful to see the data as the machine is training to gain insight in how the machine learns over time.

**Comment by perara** [comment]
Will look into implementing similar interface as seen in keras/pytorch
